resolver/addressable_resolver_test.go

```diff
@@ -22,18 +22,28 @@ import (
 	"fmt"
 	"testing"
 
+	_ "knative.dev/pkg/client/injection/kube/informers/core/v1/service/fake"
```

Comment:
should this be moved to the the `knative.dev/pkg` imports?

Reply:
I couldn't find a common style for side-effect imports across other packages, so I used this [readme](https://github.com/knative/pkg/blob/main/injection/README.md#testing-controllers) as a source of truth.